src/services/development.ts

```diff
@@ -1,36 +1,102 @@
-import { TOKENS as $, services as prodServices } from './production'
-import { merge, build, ServiceDefinition } from './utils'
-import { mocks } from '@/api/mocks'
+import { setupWorker, RestHandler, MockedRequest } from 'msw'
+
+import { TOKENS as PROD_TOKENS, services as prodServices } from './production'
+import { merge, build, ServiceDefinition, token, get } from './utils'
+import { useBootstrap } from '../index'
+import { mocks, setupHandlers } from '@/api/mocks'
+import { fs, fakeApi } from '@/api/mocks/index'
 import CookiedEnv from '@/services/env/CookiedEnv'
-import KumaApi from '@/services/kuma-api/KumaApi'
-import { mockApi } from '@/services/kuma-api/MockKumaApi'
 import Logger from '@/services/logger/DatadogLogger'
 import { disabledLogger } from '@/services/logger/DisabledLogger'
+import type { TokenType } from '@/services/utils'
 
 export { constant, get, container, createInjections, build, merge } from './utils'
-export { TOKENS } from './production'
 
-export const services: ServiceDefinition[] = merge(prodServices, [
+type Msw = ReturnType<typeof setupWorker>
+const $ = {
+  ...PROD_TOKENS,
+  msw: token<Msw>('msw'),
+  mswKumaHandlers: token<RestHandler[]>('msw.kuma.handlers'),
+  mswKumaMocks: token<RestHandler[]>('msw.kuma.mocks'),
+  mswHandlers: token<RestHandler[]>('msw.handlers'),
+}
+
+type Env = TokenType<typeof $.Env>
+export const services: ServiceDefinition[] = [
+
   [$.Env, {
     service: CookiedEnv,
     arguments: [
       $.EnvVars,
     ],
   }],
-  [$.mocks, {
-    constant: mocks,
+
+  [$.logger, {
+    service: disabledLogger(Logger),
+    arguments: [
+      $.Env,
+    ],
   }],
-  [$.api, {
-    service: mockApi(KumaApi),
+
+  // Mock Service Worker
+  [$.msw, {
+    service: (handlers: RestHandler[]) => {
+      const worker = setupWorker(...handlers)
```

Comment:
**Suggestion**: Please keep this message. It helped me several times during debugging the mock setup.

```suggestion
      const worker = setupWorker(...handlers)

      console.warn(
        '%c ✨You are mocking api requests.',
        'background: gray; color: white; display: block; padding: 0.25rem;',
      )
```

Reply:
Ah sorry missed that other `warn`, I've added this back in